### DISCUSSION TITLE
Collapse column problems

### DISCUSSION THREAD

**Discussion by marttheis** [rappasoft/laravel-livewire-tables#960] [Q&A]
Hi,

I have several issues with the new "collapse column feature".

#1.  The + and - Icons become out-of-sync when the table refreshes while the collapsed column is shown. This can be seen on the screenshot for the first two rows.
#2.  The collapsed column is not added as a full-with table row but only spans the first two columns. Doc says it should be full-width.
#3. The first column which contains the + and - icon has a head cell which  shows a (not so pretty) background color which can't be styled. It can't be addressed via table tr th:first-child{}

I would love to use this feature to show actions buttons right below the respective rows. But the issues makes this impossible. I also believe that point #1 above is actually a bug!

<img width="1144" alt="Bildschirmfoto 2022-10-27 um 23 53 20" src="https://user-images.githubusercontent.com/114695786/198405381-a2fe3e16-2f3a-4f6d-877e-b2126a815885.png">

I suppose that I could publish specific vendor views but update frequency is high and I want to get those updates later on.

I hope to get some help.

best regards
Martin

**Comment by acerjak** [comment]
I've also noticed this behavior if I have different tables on the same blade that both utilize the `collapseOnMobile`. They both open and close when one is clicked and the icons become out of sync. 

<img width="107" alt="image" src="https://user-images.githubusercontent.com/62491401/200671688-993cd9a3-b4e8-4f3e-b86d-1846a2e2fc71.png">
